themes/classic/templates/_partials/form-field.tpl

```diff
@@ -36,30 +42,60 @@
       {elseif $field.type === 'checkbox'}
 
         <span class="custom-checkbox">
-          <input type="checkbox" value="{$field.value}" name="{$field.name}" {if $field.required}required{/if}>
+          <input name="{$field.name}" type="checkbox" value="{$field.value}" {if $field.required}required{/if}>
           <span><i class="material-icons checkbox-checked">&#xE5CA;</i></span>
           <label>{$field.label}</label>
         </span>
 
       {elseif $field.type === 'date'}
 
-        <input type="text" class="form-control" placeholder="MM/DD/YYYY">
+        <input class="form-control" type="text" placeholder="MM/DD/YYYY">
         <span class="form-control-comment">
           {l s='(Ex.:31/05/1970)'}
         </span>
 
       {elseif $field.type === 'email'}
 
-        <input type="email" class="form-control" value="{$field.value}" name="{$field.name}" {if $field.required}required{/if}>
+        <input
+          class="form-control"
+          name="{$field.name}"
+          type="email"
+          value="{$field.value}"
+          {if $field.required}required{/if}>
 
       {elseif $field.type === 'password'}
 
-        <input type="password" pattern=".{literal}{{/literal}5,{literal}}{/literal}" class="form-control" value="{$field.value}" name="{$field.name}" {if $field.required}required{/if}>
-
+        <div class="input-group js-parent-focus">
+          <input
+            class="form-control js-child-focus js-visible-password"
+            name="{$field.name}"
+            type="password"
+            value="{$field.value}"
+            pattern=".{literal}{{/literal}5,{literal}}{/literal}"
```

Comment:
Nice ! I didnt know these tags.

I'd like to pass the pattern in the `$field` array so I was thinking this should become something like `{literal}{$field.pattern}{/literal}`.